src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -373,6 +384,45 @@ HMICapabilitiesImpl::~HMICapabilitiesImpl() {
   delete prerecorded_speech_;
 }
 
+bool HMICapabilitiesImpl::is_hmi_capabilities_initialized() const {
```

Comment:
@LitvinenkoIra Where this method does SDL use? You return this method but as far as I remember this method was removed as unused.